jest.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  roots: ['<rootDir>/src'],
+  testMatch: [
+    '**/__tests__/**/*.+(ts|tsx|js)',
+    '**/?(*.)+(spec|test).+(ts|tsx|js)',
+  ],
+  transform: {
+    '^.+\\.(ts|tsx)$': 'ts-jest',
+    '.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/jest-config/file-mock.js',
+    '.(css|less)$': '<rootDir>/jest-config/style-mock.js',
+  },
```

Comment:
I suggest we use `identity-obj-proxy `for mocking css imports, as [described](https://jestjs.io/docs/en/webpack#mocking-css-modules) by jest.
```suggestion
  },
    moduleNameMapper: {
    '\\.(css|less)$': 'identity-obj-proxy',
  },
```

Reply:
done